grpc/src/main/java/com/linecorp/armeria/server/grpc/FramedGrpcService.java

```diff
@@ -285,8 +285,10 @@ private <I, O> void startCall(
         final MethodDescriptor<I, O> methodDescriptor = methodDef.getMethodDescriptor();
         final Executor blockingExecutor;
         if (useBlockingTaskExecutor || registry.needToUseBlockingTaskExecutor(methodDef)) {
+            ctx.setShouldUseBlockingTaskExecutor(true);
```

Comment:
Should we use an attribute instead of adding a public API? Because if  `shouldUseBlockingTaskExecutor()` is added to `ctx`, we also need to check all implementations to respect the value.
```suggestion
            ctx.setAttr(GRPC_USE_BLOCKING_EXECUTOR, true);
```

Reply:
Sorry for the delay 😓 

I have updated it 👍